salesforce/10.9/modules/ROOT/pages/salesforce-connector-reference.adoc

```diff
@@ -0,0 +1,5130 @@
+updating an existing contact. IMPORTANT: if you are converting a lead into a person account, do not specify the contactId or an error will result. Specify only the accountId
+of the person account. If no contactID is specified, then the API creates a new contact that is implicitly associated with the Account. To create a new contact, the client
+application must be logged in with sufficient access rights. To merge a lead into an existing contact, the client application must be logged in with read/write access to the
+specified contact. The contact name and other existing data are not overwritten (unless overwriteLeadSource is set to true, in which case only the LeadSource field is
+overwritten). For information on IDs, see ID Field Type. |  |
+| Account Id a| String | ID of the Account into which the lead will be merged. Required only when updating an existing account, including person accounts. If no accountID is specified, then the API
+creates a new account. To create a new account, the client application must be logged in with sufficient access rights. To merge a lead into an existing account, the client
+application must be logged in with read/write access to the specified account. The account name and other existing data are not overwritten. For information on IDs, see ID
+Field Type. |  |
+| Send Email To Owner a| Boolean | Specifies whether to send a notification email to the owner specified in the ownerId (true) or not (false, the default). | false |
+| Opportunity Name a| String | Name of the opportunity to create. If no name is specified, then this value defaults to the company name of the lead. The maximum length of this field is 80 characters. If
+doNotCreateOpportunity argument is true, then no Opportunity is created and this field must be left blank; otherwise, an error is returned. |  |
+| Do Not Create Opportunity a| Boolean | Specifies whether to create an Opportunity during lead conversion (false, the default) or not (true). Set this flag to true only if you do not want to create an opportunity
+from the lead. An opportunity is created by default. | false |
+| Over Write Lead Source a| Boolean | Specifies whether to overwrite the LeadSource field on the target Contact object with the contents of the LeadSource field in the source Lead object (true), or not (false,
+the default). To set this field to true, the client application must specify a contactId for the target contact. | false |
+| Opportunity Id a| String | The ID of an existing opportunity to relate to a lead during lead conversion. |  |
+| Owner Id a| String | Name associated with the owner ID that owns this lead |  |
+| Related Person Account Id a| String | When converting a lead to a business account and an existing person account, this value specifies the ID of the person account. |  |
```

Comment:
```suggestion
| Related Person Account Id a| String | When converting a lead to a business account for an existing person account, this value specifies the ID of the person account. |  |
```

Reply:
?

Reply:
Same ? for line 4405

Reply:
I think it is correct as is even though it seems funny.